test/testcomplexity.cpp

```diff
@@ -0,0 +1,128 @@
+
+#include "abc.h"
+#include "constant.h"
+#include "power.h"
+#include "sum.h"
+#include "product.h"
+#include "trigonometric.h"
+#include "tsymtests.h"
+#include "undefined.h"
+#include "logarithm.h"
+
+using namespace tsym;
+
+TEST_GROUP(Complexity)
+{
+    
```

Comment:
The variables declared in the following lines are visible in all test cases, which isn't necessary. If you need one defined in the whole test group, it's fine to have it as a class variable, otherwise declare it locally to reduce the scope of the variable.